src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -309,14 +309,14 @@ private void CheckDisposed()
         // a pending operation, it would cause random failures in the other threads when we expect a valid handle.
         private void CancelPendingResponseStreamReadOperation()
         {
-            WinHttpTraceHelper.Trace("WinHttpResponseStream.CancelPendingResponseStreamReadOperation");
+            if (NetEventSource.IsEnabled) NetEventSource.Enter(this);
```

Comment:
Do we need a `NetEventSource.Exit` for this function?

Reply:
For this example, yes, I'll add an Exit() since I added the Enter().  In general, though I wasn't planning on doing a lot of changes/additions to the logging other than the conversion to NetEventSource.  More logging might be done in a future PR.